### DISCUSSION TITLE
Export Group Data

### DISCUSSION THREAD

**Discussion by xyz-333** [oss-apps/split-pro#125] [Ideas]
Hi, I've been using various expense sharing apps but they all either lack some functionality or become unusable (such as splitwise).
The one thing I am missing in SplitPro is an option to export group expenses, which could be as simple as .csv file.
I assumed it could be extracted from the .json generated through "Download splitpro data", but it does not seem to store individual expenses, which is essential for me.
Given how many projects such as SplitPro exist and that they are in fact very similar without any revolutionary functionalities, it would be great if there's some standard, a shared format for storing the data that would allow to change the apps without losing everything.

**Comment by KMKoushik** [comment]
we added export option for friends section here https://github.com/oss-apps/split-pro/pull/123 and can add it for groups as well.

also we can add the the expenses to the Download splitpro data option. i think this is important so splitpro won't lock it's users.

also, splitpro exist to give an opensource way to manage expenses and not to lock you down. once you have the above data you can move to any product that supports API to add expense. we are also planning to add it in our end!

**Reply by Golllog** [reply]
Fantastic App.  Love the work so far. Splitwise has become unusable and the other alternatives, you give up too much privacy and/or they don't have the features we need.  

Would love the option to export data from a group perspective, not limited to a friend rather the whole groups data regardless of who paid what etc.

Appreciate this is a work in progress so thank you!

**Reply by Golllog** [reply]
Hi @KMKoushik sorry to continue here, I wanted to know do I need to add this as a Feature Request to the "Issues" section.  To me, this is a major requirement, I tend to work with people and create a lot of short term groups for different things, e.g Holiday. Once the holiday is over, settle with anyone involved and archive or delete it.  Without the ability to extract the data just for the group, I can't do this.

This isn't a complaint I appreciate all that's been done by the wonderful contributors so far, just explaining, and my question is to get this in the list for work/coding, do I need to add it to the "Issues" section?

Kind thanks.

**Reply by krokosik** [reply]
Yeah, please submit an issue :)
